packages/mdc-ripple/README.md

```diff
@@ -32,7 +32,7 @@ path: /catalog/ripples/
     - [Keyboard interaction for custom UI components](#keyboard-interaction-for-custom-ui-components)
     - [Specifying known element dimensions](#specifying-known-element-dimensions)
   - [Caveat: Edge](#caveat-edge)
-  - [Caveat: Safari](#caveat-safari)
+  - [Caveat: Safari < 10](#caveat-safari)
```

Comment:
Update this link to #caveat-safari--10 so it doesn't break with the updated heading

Reply:
Good catch! I added a backward compatible anchor tag (`<a name="caveat-safari"></a>`) to avoid breaking existing bookmarks.

Reply:
@acdvorak I ran the generator and it produced exactly the same thing as Github (caveat-safari--10). So I think both you and Ken's should work.